hypha/apply/funds/tables.py

```diff
@@ -443,3 +443,18 @@ class Meta:
         order_by = ('-created_at',)
         attrs = {'class': 'all-reviews-table'}
         empty_text = _('No reviews available')
+
+
+class StaffAssignmentsTable(tables.Table):
+    full_name = tables.LinkColumn('funds:submissions:list', orderable=True, verbose_name="Staff", attrs={'td': {'class': 'title'}})
```

Comment:
@Parbhat I would like to add a query to this linked column so the view gets filtered on the reviewer.

`…/apply/submissions/all/?reviewers=123`

Reply:
or easier to just create a separate view for this?

Reply:
For filtering, we can use `django-filters`'s FilterView like https://github.com/OpenTechFund/hypha/blob/32679916c2f9e5e82322296b14966659b5d708b9/hypha/apply/funds/views.py#L1009